src/psyclone/nemo.py

```diff
@@ -492,6 +490,18 @@ def view(self, indent=0):
         print(self.indent(indent) + self.coloured_text + "[" +
               self.ktype + "]")
 
+    @property
+    def ast(self):
```

Comment:
Is this actually used anywhere? I presume so but can't find it.

Reply:
Yes, it's needed by the base class(es) that assume a Kernel has an `ast` property.